projects/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
 
-# Create your models here.
+from companies.models import CompanyProfile
+
+
+PROJECT_STATUSES = [
+    ("active", "Active"),
+    ("completed", "Completed"),
+]
+
+class Project(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    status = models.CharField(choices=PROJECT_STATUSES, blank=False)
+    information = models.TextField()
+    required_funding = models.DecimalField(max_digits=15, decimal_places=2, blank=False, null=False)
+    raised_amount = models.DecimalField(max_digits=15, decimal_places=2, blank=True, null=True)
```

Comment:
Consider adding a constraint to prevent `raised_amount` > `required_funding`
You can add a custom validation method in the model or use a `CheckConstraint` to ensure logical consistency.
```
from django.core.exceptions import ValidationError

def clean(self):
    if self.raised_amount and self.raised_amount > self.required_funding:
        raise ValidationError("Raised amount cannot exceed required funding.")
```
Or with a `CheckConstraint`:
```
constraints = [
    models.CheckConstraint(
        check=models.Q(raised_amount__lte=models.F('required_funding')),
        name='raised_amount_not_exceed_required'
    )
]
```